PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -91,35 +91,23 @@ def wait_for_step(step,time):
 
 @step(r'the "([^"]+)" parallel call should return with:')
 def compare_thread_return(step,apiCall):
-    #Prepare response list for comparison
+    """Prepare response list for comparison"""
     logger.debug(world.responses)
     future_results = world.config['future_results'][apiCall]
 
     for result in future_results:
         response_list = pool.fetch_results(result,1)
 
-        #Exclude duration from response list
+        '''Exclude duration from response list'''
```

Comment:
Another question:
replaceing `#` -> with `'''`, why did you find that this is needed. Didn't see anything about in PEP 257...

Reply:
IntelliJ was suggesting I use triple quotes, I wasn't really sure why but I thought i'd switch it over anyways, i'll just switch it back.